src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ConfigurationGeneral.xaml

```diff
@@ -77,6 +77,8 @@
     </EnumProperty>
     <StringListProperty Name="AvailablePlatforms" Separator="," />
     <BoolProperty Name="Optimize" Description="Should compiler optimize output?" />
+    <!-- Same as MSBuildProjectDirectory but has a directory separator '\' appended at the end.-->
+    <StringProperty Name="ProjectDir" Visible="false"/>
```

Comment:
Instead of using a property that might not actually exist (it's not built-in), why can you just append the backslash to the MSBuildProjectDirectory?